### ISSUE TITLE
differentiate marginal additions to main from commentaries

### ISSUE THREAD

**Issue by dstoekl** [SegmOnto/Guidelines#46]
In frequent cases, marginal additions/corrections to/from the main text can be distinguished from marginal comments or full blown commentaries around some hypotext. I would therefore propose to add a CommentaryZone.

**Comment by dstoekl** [comment]
eg. 
![image](https://user-images.githubusercontent.com/39535215/145724220-f98c7ec7-6101-4df2-ac1c-452df1eb2abb.png)

**Comment by Jean-Baptiste-Camps** [comment]
I would suggest defining this as a suggested `subtype` of `MarginText`.

**Comment by gabays** [comment]
I agree, except the name is `MarginTextZone` (cf. [here](https://github.com/SegmOnto/Guidelines/tree/main/zones/MarginTextZone)). So: `MarginTextZone:commentary`.
